### PR TITLE
PEX: remove workaround for krpc marshaller

### PR DESCRIPTION
**Pull request by yarikk**
anacrolix/dht#54 fixes the marshaller — the workaround can be removed.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/anacrolix/args v0.4.1-0.20211104085705-59f0fe94eb8f
 	github.com/anacrolix/chansync v0.3.0
 	github.com/anacrolix/confluence v1.9.0 // indirect
-	github.com/anacrolix/dht/v2 v2.13.0
+	github.com/anacrolix/dht/v2 v2.13.1-pr54-fix-marshal
```
Comment (yarikk) [inline go.mod:11]: This needs to be updated when anacrolix/dht#54 is merged and received a version tag. This PR can't be merged until then.

**Comment by anacrolix** [comment]
See https://github.com/anacrolix/dht/issues/33#issuecomment-989531282.

**Comment by yarikk** [comment]
rebased, updated go.mod

**Review by anacrolix — APPROVED** [review]
Merge at will.

**Comment by yarikk** [comment]
Let's hold until #693 is landed.

**Pull request merged**